sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Generated/Samples/Samples_ConversationAnalysisClient.cs

```diff
@@ -52,7 +52,7 @@ public void Example_AnalyzeConversation()
 
         [Test]
         [Ignore("Only validating compilation of examples")]
-        public void Example_AnalyzeConversation_AllParameters()
+        public async Task Example_AnalyzeConversation_Async()
```

Comment:
Can we not use snake_case? This is C#, not Python. C# (and .NET in general) uses PascalCase for most public constructs like methods.